src/components/Button/index.tsx

```diff
@@ -315,7 +315,7 @@ function Button(
                     large ? styles.buttonLarge : undefined,
                     success ? styles.buttonSuccess : undefined,
                     danger ? styles.buttonDanger : undefined,
-                    isDisabled && (success || danger) ? styles.buttonOpacityDisabled : undefined,
+                    isDisabled ? styles.buttonOpacityDisabled : undefined,
```

Comment:
I was asked by @shawnborton to adjust the buttons styles according to [the most recent design](https://www.figma.com/file/qQIIaxVUdtggrh8eg8bI5B/Brand-Guidelines-%26-Product-UI?type=design&node-id=1-2&mode=design&t=OQj3qhSXAQDceXWT-0) (not sure if you have access to it) - to cut the story short no matter in which state is the button (default, success or danger) we should lower its opacity by 50% in case it is disabled. I don't think this change would have huge impact on the App functionality - I've clicked through the app and it seems everything is ok. Do you want something more from me here?